app/src/pages/Organization.tsx

```diff
@@ -23,7 +23,7 @@ const DIRECTORS = [
   { name: "Liam Orovec", wcaId: "2014OROV01" },
 ] as const;
 
-const WCA_PROFILE_URL = "https://www.worldcubeassociation.org/persons/";
+export const WCA_PROFILE_URL = "https://www.worldcubeassociation.org/persons/";
```

Comment:
Move this to a common file, then import it to this file too instead of exporting it from this file.